### PR TITLE
Community Survey project card

### PR DESCRIPTION
**Pull request by danyc23**

### CODE REVIEW

**Comment by jbubar** [comment]
@danyc23 This looks great! can you link to the issue and link some photos in it?

**Comment by danyc23** [comment]
Issue #932 
- The project card was successfully created.

Screen Shot 2021-02-08 at 12 05 02 PM

Screen Shot 2021-02-08 at 12 03 55 PM

Screen Shot 2021-02-08 at 12 04 27 PM

File: _projects/open-community-survey.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+      github: 'https://github.com/johnr54321'
+    picture: https://avatars.githubusercontent.com/johnr54321
+links:
+  - name: GitHub
+    url: 'https://github.com/hackforla/community-survey-nc-websites'
+  - name: Slack
+    url: 'https://hackforla.slack.com/archives/C01H0HUDMCK'
+  - name: Getting Started
+    url: 'https://github.com/hackforla/community-survey-nc-websites/projects/1#card-51519391'
+
```
Comment (ExperimentsInHonesty) [inline _projects/open-community-survey.md]: There is a blank line here and a few other places in the file.  I think the convention is to not have any blank lines.  If we have changed that, or you are copying your format from another page that does have blank lines, please reply with that detail.

File: _projects/open-community-survey.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+looking:
+  - category: PM
+    skill: Product Manager
+  - category: PM
+    skill: Product Owner
+  - category: Data
+    skill: Data Scientists
+
```
Comment (ExperimentsInHonesty) [inline _projects/open-community-survey.md]: blank line

File: _projects/open-community-survey.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+location:
+  - Los Angeles
+  - Remote
+
```
Comment (ExperimentsInHonesty) [inline _projects/open-community-survey.md]: blank line

File: _projects/open-community-survey.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+      slack: 'https://hackforla.slack.com/team/UE1UG1YFP'
+      github: 'https://github.com/ExperimentsInHonesty'
+    picture: https://avatars.githubusercontent.com/ExperimentsInHonesty
+  - name: Phoebe Ng
+    role: UX Researcher
+    links:
+      slack: 'https://hackforla.slack.com/team/U01GTRX0H7X'
+      github: 'https://github.com/phoebeng'
+    picture: https://avatars.githubusercontent.com/phoebeng
+  - name: John
```
Comment (ExperimentsInHonesty) [inline _projects/open-community-survey.md]: Can you remove John.  He has stepped down .  I'll make a note on the issue too.

**Review by ExperimentsInHonesty — CHANGES_REQUESTED** [review]
See my individual comments for changes

**Comment by danyc23** [comment]
Changes are made and committed.

Screen Shot 2021-02-08 at 5 18 40 PM

**Review by drubgrubby — CHANGES_REQUESTED** [review]
The Open Community Survey card looks great. Nice work.  

In looking at the "Files changed" it looks like there are also changes made to tdm-calculator.md, which is not part of this PR.  I'm guessing that this is because you didn't change back to gh-pages before creating a new branch for this issue.  We need to remove the changes to tdm calculator before we push this.  If you have questions about how to do that, we can go over it at the meeting.

**Comment by drubgrubby** [comment]
Also, it should say 
```
Fixes #932
``` 
so that the issue is automatically closed when the PR is merged.

**Pull request closed without merging**